lib/media/streaming_engine.js

```diff
@@ -2345,7 +2345,7 @@ shaka.media.StreamingEngine = class {
 
     const lastDiscontinuitySequence =
         mediaState.lastSegmentReference ?
-            mediaState.lastSegmentReference.discontinuitySequence : null;
+            mediaState.lastSegmentReference.discontinuitySequence : -1;
```

Comment:
-1 is the defautl value on https://github.com/shaka-project/shaka-player/blob/main/lib/media/segment_reference.js#L314